examples/jiralert.yml

```diff
@@ -30,7 +30,6 @@ receivers:
     project: AB
     # Copy all Prometheus labels into separate JIRA labels. Optional (default: false).
     add_group_labels: false
-
```

Comment:
I would keep this newline  for readability

Reply:
kept the newline.